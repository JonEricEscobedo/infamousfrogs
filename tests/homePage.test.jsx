import React from 'react';
import {shallow, mount} from 'enzyme';
import toJson from 'enzyme-to-json'
import ReactTestUtils from 'react-dom/test-utils'; 
import homePage from '../react-client/src/viewComponents/homePage.jsx';


jest.dontMock('../react-client/src/viewComponents/homePage.jsx');

test('contains spec with an expectation', function() {
  expect(true).toBeTruthy();
});

test('homePage component should render as expected', () => {
  const component = shallow(<homePage />)
  const tree = toJson(component)

  expect(tree).toMatchSnapshot()
})

test('should be a stateful class component', function() {
  expect(React.Component.isPrototypeOf(homePage)).toBeTruthy();
});

