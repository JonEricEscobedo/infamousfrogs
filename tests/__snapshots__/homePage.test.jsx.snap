// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homePage component should render as expected 1`] = `<homePage />`;
